PrimerSDK/Services/API/Models/CreateAccountRequestParams.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  CreateAccountRequest.swift
+//  primer-checkout-api
+//
+//  Created by Evangelos Pittas on 26/2/21.
+//
+
+import Foundation
+
+struct CreateAccountRequestParams {
```

Comment:
τι είναι αυτό?

Reply:
Nice catch!! Removed it, merged from develop and QA it. All seem to work.